style/preview.less

```diff
@@ -286,6 +286,51 @@
 		}
 	}
 
+	&-offline {
+		height: 285px;
+
+		&-body {
+			height: 100%;
+			display: flex;
+			flex-direction: column;
+			justify-content: center;
+
+			&-message {
+				display: flex;
+				margin-left: 16px;
+				margin-right: 16px;
+				font-size: 16px;
+				font-weight: normal;
+				font-stretch: normal;
+				font-style: normal;
+				line-height: 1.4;
+				letter-spacing: normal;
```

Comment:
All those `<something>: normal` may not be needed.

Same with the &-retry block below.